frontend/src/app/navbar/navbar.component.html

```diff
@@ -1,7 +1,7 @@
 <nav class="navbar navbar-expand-md bg-dark py-0" data-bs-theme="dark">
   <div class="container-fluid">
     <span class="navbar-brand">
-      <img src="assets/logo.png" alt="" />
+      <img src="assets/logo.png" alt="Logo" />
```

Comment:
I think I chose to leave it blank because it's purely decorative and blind people don't give a damn about it